bin/plugin/open/selfListSessions

```diff
@@ -107,7 +107,7 @@ else {
     my $list = $fnret->value;
 
     my @result;
-    foreach my $id (sort { $list->{$b} <=> $list->{$a} } keys %{$list}) {
+    foreach my $id (sort { $list->{$b}->{'timestamp'} <=> $list->{$a}->{'timestamp'} } keys %{$list}) {
```

Comment:
better off using `{'id'}`, it's guaranteed to be monotonically increasing (primary key)